scripts/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.3"
+services:
+  hello:
+    image: busybox
+    command:
+      sh -c 'while true; do nc -lp 8000 -e echo -e "HTTP/1.1 200 OK\r\n\r\n{\"msg\":\"hello\",\"host\":\"$$HOSTNAME\"}"; done'
```

Comment:
Co to za opcja -e w nc ?

Reply:
W sumie to nie wiem. Jakim cudem to działa? :D

Reply:
U mnie na WSL nie działa :(

Reply:
Ok, unixowy netcat obsługuje `-e` i służy ono to podania programu do wykonania - w tym wypadku `echo`.
Źródło: https://www.commandlinux.com/man-page/man1/nc.1.html

Reply:
Jbc to zastąpiłem pipem żeby było bardziej uniwersalnie.